### ISSUE TITLE
is IPVPN alive somewhere else?  I'd like to try it out but Quick Start commands fail on Ubuntu 20.04

### ISSUE THREAD

**Issue by bmullan** [xaionaro-go/ipvpn#14]
I know IPFS has improved alot in the last 2 years and I was curious how IPVPN worked using IPFS to help auto-build its
mesh VPN.

> $ GO111MODULE=on go get github.com/my-network/ipvpn/cmd/ipvpnd
> go get: github.com/my-network/ipvpn@none updating to
> 	github.com/my-network/ipvpn@v0.0.0-20190907182739-d144cd8c479e requires
> 	github.com/ipfs/go-ipfs@v0.4.22-0.20190904220133-06153e088e2b requires
> 	github.com/golangci/golangci-lint@v1.17.1 requires
> 	github.com/go-critic/go-critic@v0.0.0-20181204210945-1df300866540: invalid pseudo-version: does not match version-control timestamp (expected 20190526074819)

**Comment by xaionaro** [comment]
First of all I have to say, that I haven't really completed the project. And it worked very unstable last time I checked. I did somehow make it so-so work at some moment, but then I started a redesign to make it stable, and never finished :(
And I really hope I will finish it one day.

About how to build it: if you will replace `go1.13beta1` with a path to a go1.13 compiler in `Makefile`, then just `make` should work. It appears it does not execute when compiled with go1.16 -- I will fix it.
```
GO111MODULE=off go get -d github.com/my-network/ipvpn/cmd/ipvpnd
cd "$(go env GOPATH)"/src/github.com/my-network/ipvpn
vim Makefile
make
```
Also I'm glad to explain the design of how IPFS is used if that was the question :)

If the project is relevant to you, I can take a look where I stopped on this weekend to understand how much is missing.

**Comment by mrbluecoat** [comment]
I agree, this project has a lot of potential. You should give it a refresh.

**Comment by mrbluecoat** [comment]
For those who find this post, [Hyprspace](https://github.com/hyprspace/hyprspace) appears to be the successor to this concept

**Comment by bmullan** [comment]
@mrbluecoat 
I've used Hyprspace.... But it's problem is it does not use wireguard so performance isn't that great.
That's why I wanted to see if ipvpn. Improve performance because it uses wire guard
